integrations/server/test_covidcast.py

```diff
@@ -308,26 +307,28 @@ def fetch(geo_value):
       return response
 
     # test fetch a specific region
-    r = fetch('11111')
+    r = fetch(MSA[0])
     self.assertEqual(r['message'], 'success')
     self.assertEqual(r['epidata'], expected[0:1])
     # test fetch a specific yet not existing region
-    r = fetch('55555')
-    self.assertEqual(r['message'], 'no results')
+    r = fetch('11111')
+    self.assertEqual(r['message'], 'Invalid geo_value(s) 11111 for the requested geo_type msa')
     # test fetch multiple regions
-    r = fetch('11111,22222')
+    r = fetch('{},{}'.format(MSA[0], MSA[1]))
     self.assertEqual(r['message'], 'success')
     self.assertEqual(r['epidata'], expected[0:2])
     # test fetch multiple noncontiguous regions
-    r = fetch('11111,33333')
+    r = fetch('{},{}'.format(MSA[0], MSA[2]))
```

Comment:
nit: shorter, potentially easier to read/understand
```suggestion
    r = fetch(f'{MSA[0]},{MSA[2]}'
```